generate-web/success/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Registration successful</title>
```

Comment:
Email validation is done at a later stage (?), I'm not sure why this is part of `generate-web` UI.

Reply:
both `/success/` and `/error/` are parts of quickstart.holo.host. Either we keep them in this repo for logical reasons or we can separate entire `/generate-web` folder into a quickstart repo and use `/core` as a submodule.

Reply:
Let's have a call, I wonder if there's a disagreement about how this should look like.

Reply:
moving out to hpstatus repo